.gitignore

```diff
@@ -39,3 +39,5 @@ integration-coverage.html
 go.work*
 
 /result
+# File copied into mdatagen to be embedded
+cmd/mdatagen/.distributions.yaml
```

Comment:
Why copy/ignore vs not having it here in from the beginning?

Reply:
the file was already present in the root of the repository. There is a go:generate trick used to copy the file in the work folder:
`//go:generate cp -r ../../distributions.yaml .distributions.yaml`

This trick doesn't work with `go install` though. Yes, we could just keep it around in git instead, you're right.

Reply:
OK, this should be good now.